frontend/src/container/ListOfDashboard/DashboardsList.tsx

```diff
@@ -455,7 +455,9 @@ function DashboardsList(): JSX.Element {
 									alt="dashboard-image"
 								/>
 								<Typography.Text data-testid={`dashboard-title-${index}`}>
-									{dashboard.name}
+									<Link to={`${ROUTES.ALL_DASHBOARD}/${dashboard.id}`}>
```

Comment:
nit: we have `getLink()` that gives us the link. can we use it instead of `ROUTES.ALL_DASHBOARD}/${dashboard.id}`